.ado/pipelines/templates/jobs-configuration.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
       inlineScript: |
 
         # load json data from downloaded pipeline artifact json
+        $globalInfraDeployOutput = Get-ChildItem $(Pipeline.Workspace)/terraformOutputGlobalInfra/*.json | Get-Content | ConvertFrom-JSON
```

Comment:
I thought I was using `steps-parse-terraform-output.yaml` for the global terraform values instead. Have you dropped it?

Reply:
yes, to keep it consistent with the other places where we use the outputs. the parse function is only used/needed if we dont use the output in a script directly

Reply:
I mean we only need `acr_name` here, but wouldn't it be a bit cleaner and easier to read to run the `steps-parse-terraform-output.yaml` at the top of each job instead?

Reply:
hmm good question. This would only work for the global infra output, not for the release unit. I'm can't say I'd prefer either way.
@msimecek thoughts from your side?

Reply:
True, the JSON format for the release unit is slightly different and needs to support 1 or more release units. Could be challenging to translate that into meaningful variables.

I'd only the global infra output in mind - we've `steps-parse-terraform-output.yaml`, we're already using it - so why not using it consistently accross all stages? `$(acr_name)` is definitely easier to read than `$($globalInfraDeployOutput.acr_login_server.value)` ;)

Reply:
yes agree, it would be much easier to read. I'm just not sure if it will be more confusing to people since we might then have two different ways to read the variables in the same scripts?